scripts/install_plugin_modules.js

```diff
@@ -99,9 +101,13 @@ function assertPackage (name, version, dependency, external) {
 function assertIndex (name, version) {
   const index = `'use strict'
 
+const fs = require('fs')
```

Comment:
Why are `fs` and `path` needed here?

Reply:
Previously, `requirePackageJson` was inlined. I'll fix that now.